packages/@aws-cdk/aws-ec2/lib/security-group-rule.ts

```diff
@@ -234,7 +234,92 @@ export class TcpAllPorts implements IPortRange {
 }
 
 /**
- * All TCP Ports
+ * A single UDP port
+ */
+export class UdpPort implements IPortRange {
+  public readonly canInlineRule = true;
+
+  constructor(private readonly port: number) {
+  }
+
+  public toRuleJSON(): any {
+    return {
+      ipProtocol: Protocol.Udp,
+      fromPort: this.port,
+      toPort: this.port
+    };
+  }
+
+  public toString() {
+    return `${this.port}`;
```

Comment:
Can you add the word UDP to this description string?

LIke `UDP ${this.port}`?

Reply:
@rix0rrr Sure, while I prefix all the UDP-related `toString()` to use `UDP `, should I also do the same for the TCP ones? Just for consistency. We'd achieve uniqueness either way, but thought why should one be treated as a first-class citizen over the other?

Reply:
Also, is it worth it to do something like `${Protocol.Udp} ${this.port}` as compared to `UDP ${this.port}`?